plugins/modules/ndo_fabric_span_session.py

```diff
@@ -0,0 +1,698 @@
+            - This parameter or O(destination_epg.epg.schema_id) is required.
+            type: str
+          anp:
+            description:
+            - The name of the ANP that contains the destination EPG.
+            - This parameter or O(destination_epg.epg.anp_uuid) is required.
+            type: str
+          anp_uuid:
+            description:
+            - The UUID of the ANP that contains the destination EPG.
+            - This parameter or O(destination_epg.epg.anp) is required.
+            type: str
+      destination_ip:
+        description:
+        - The destination IP address to route SPAN Session packets.
+        type: str
+      source_ip_prefix:
+        description:
+        - The destination IP address prefix to route SPAN Session packets.
+        type: str
+      span_version:
+        description:
+        - The version of the SPAN Session.
+        - Defaults to C(v2) when unset during creation.
+        choices: [ v1, v2 ]
+        type: str
+      enforce_span_version:
+        description:
+        - Enforce the SPAN Session version defined in O(destination_epg.span_version).
+        - Defaults to true when unset during creation.
+        type: bool
+      flow_id:
+        description:
+        - The flow ID of the SPAN Session packets.
+        - The value must be in the range 1 - 1023.
+        - Defaults to 1 when unset during creation.
+        type: int
+      ttl:
+        description:
+        - The time to live (TTL) of the SPAN Session packets.
+        - The value must be in the range 1 - 1023.
+        - Defaults to 1 when unset during creation.
+        type: int
+      dscp:
+        description:
+        - The DSCP value for sending the monitored SPAN Session packets.
+        - Defaults to C(unspecified) when unset during creation.
+        choices:
+          - af11
+          - af12
+          - af13
+          - af21
+          - af22
+          - af23
+          - af31
+          - af32
+          - af33
+          - af41
+          - af42
+          - af43
+          - cs0
+          - cs1
+          - cs2
+          - cs3
+          - cs4
+          - cs5
+          - cs6
+          - cs7
+          - expedited_forwarding
+          - voice_admit
+          - unspecified
+        type: str
+  destination_port:
+    description:
+    - The destination port configuration group.
+    - This parameter or O(destination_epg) or O(destination_port_channel) is required when creating the SPAN Session.
+    type: dict
+    suboptions:
+      port_uuid:
+        description:
+        - The UUID of the destination port to use for the SPAN Session.
+        - This parameter or O(destination_port.port) is required when creating the SPAN Session.
+        type: str
+      port:        
+        description:
+        - The destination port to use for the SPAN Session.
+        - This parameter or O(destination_port.port_uuid) is required when creating the SPAN Session.
```

Comment:
Since the "required when creating the SPAN Session" statement is specified in the parent options I don't think we need to state it again in the sub options. 
```suggestion
        - This parameter or O(destination_port.port) is required.
        type: str
      port:        
        description:
        - The destination port to use for the SPAN Session.
        - This parameter or O(destination_port.port_uuid) is required.
```

Reply:
Done!